setup.py

```diff
@@ -24,7 +24,7 @@ def package_setup():
 #     Install Katana native library
 # conda install -c katanagraph/label/dev -c conda-forge katana-cpp=={{KATANA_VERSION}}
 #     Install Katana Python Conda dependencies (due to problems in the pip pyarrow package)
-# conda install -c conda-forge pyarrow==4.0.1
+# conda install -c conda-forge python==3.8 pyarrow==4.0.1
```

Comment:
Don't we use 3.9?

Reply:
All our packages are still built for 3.8. We probably need to be building both, but we don't for the moment. Dev environments are often 3.9.